redaxo/src/addons/cronjob/package.yml

```diff
@@ -12,6 +12,11 @@ page:
         cronjobs: { title: 'translate:title' }
         log: { title: 'translate:log' }
 
+pages:
+    system/log/cronjob:
+        title: Cronjob
+        perm: admin[]
+
 requires:
     redaxo: ^5.6.2
```

Comment:
hier muss es nun 5.7.0 heißen, oder?

Reply:
Ohne 5.7 würde das feature nicht funktionieren, aber grundsätzlich notwendig ist 5.7 vermutlich nicht, oder?

Reply:
Aber der Page-Eintrag oben erzeugt eine Subpage, die in 5.6 nicht funktioniert.

Reply:
Done